### DISCUSSION TITLE
GOPRO 11 NAKED Flywoo v1 GOPRO LABS 02_03_70

### DISCUSSION THREAD

**Discussion by simoneenomis** [gopro/labs#1053] [General]
Hi everyone, thanks to anyone who can help me... so my gopro naked 11 Hero only records in 30 fps max if I try to set it to 4K 60 fps or 5.3K 60 fps the recording starts and stops, I'll start by saying that I've tried any SD card high speed seems like it can't handle 60 fps....Help

**Comment by dnewman-gpsw** [comment]
These is no common reason.  Try a !RESET to clear Labs settings, so clear on all BITR extensions as that can be an issue at 60. Higher fps needs lower bitrates.  Use 160Mb/s maximum for 60fps.

**Reply by simoneenomis** [reply]
I solved it by doing a replacement of the Flywoo tanks modification flag
